srcopsmetrics/iterator.py

```diff
@@ -84,7 +84,7 @@ def run(self):
                     wait_time = gh_time - local_time.replace(tzinfo=None)
```

Comment:
I think it makes a little more sense to add the 60 seconds here instead.

Reply:
Also might be good to have an env variable to tune that time if you want?